### ISSUE TITLE
SAM lambda functions receive list-based tags

### ISSUE THREAD

**Issue by kristofwillen** [kristofwillen/cfntagger#25]
Tags for AWS::Serverless::Function based resources have their tags converted to lists by cfntagger. They should remain map based:

```bash
$ cfntagger -f myfile.yml
$ git diff
       Tags:
-        "Application": foo
+      - Key: "Application"
+        Value: foo
```

**Comment by kristofwillen** [comment]
Still issue in 0.10.2: now no tags are added!

**Comment by kristofwillen** [comment]
Fixed in v0.10.3
